File: online-judge-solutions/Codeforces/32B.py
#https://codeforces.com/problemset/problem/32/B
#Lang		:	Python 3.8
#Time		:	92 ms
#Memory	:	0 KB
s=input()
while(len(s)!=0):
    if(s[0]=='.'):print('0',end="")
    else:
        print('1',end="") if s[1]=='.' else print('2',end="")
        s=s[1:]
    s=s[1:]
print('\n')

File: online-judge-solutions/Codeforces/32B.cpp
//https://codeforces.com/problemset/problem/32/B
//Lang		:	GNU C++14
//Time		:	30 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	string s;
	while(cin>>s){
		while(s.size()!=0){
			if(s[0]=='.'){cout<<0;}
			else{
				s[1]=='.'?cout<<1:cout<<2;
				s.erase(s.begin());
			}
			s.erase(s.begin());
		}cout<<endl;
	}
	return 0;
}
